src/extension/sldService/src/test/java/org/geoserver/sldservice/TestDataWriter.java

```diff
@@ -0,0 +1,33 @@
+package org.geoserver.sldservice;
```

Comment:
Missing copyright header.

Reply:
Doh, this class I wrote to prepare testing data but was not meant to be committed, will remove.